### PR TITLE
Delete Magicseaweed

### PR DESCRIPTION
**Pull request by gjohansson-ST**
## Proposed change

Remove Magicseaweed integration as deprecation was done #25661

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [x] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/90277
- Link to parent pull request in the Brands repository: https://github.com/home-assistant/brands/pull/4239
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Review by cgarwood — APPROVED** [review]

File: source/_integrations/magicseaweed.markdown [hunk 1]
```diff
@@ -1,92 +0,0 @@
----
-title: Magicseaweed
```
Comment (frenck) [inline source/_integrations/magicseaweed.markdown:2]: There is no redirect added for this missing/removed page.

Please add one.

**Review by frenck — COMMENTED** [review]
Please address this review comment in a new PR @gjohansson-ST 

CC @cgarwood

File: source/_integrations/magicseaweed.markdown (hunk 1 above)
Comment (gjohansson-ST) [reply source/_integrations/magicseaweed.markdown:2]: Sorry, missed it
#26726

Comment (frenck) [reply source/_integrations/magicseaweed.markdown:2]: Thanks 👍

**Pull request merged**